package.json

```diff
@@ -9,23 +9,23 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/pa11y/ci.git"
+    "url": "https://github.com/pa11y/pa11y-ci.git"
   },
-  "homepage": "https://github.com/pa11y/ci",
-  "bugs": "https://github.com/pa11y/ci/issues",
+  "homepage": "https://github.com/pa11y/pa11y-ci",
+  "bugs": "https://github.com/pa11y/pa11y-ci/issues",
   "license": "LGPL-3.0-only",
   "engines": {
     "node": ">=12"
```

Comment:
Does having an upper limit here feel appropriate now?

Reply:
Updated to `"^12.0.0 || ^14.0.0 || ^16.0.0"`

Reply:
For me, and probably some others, this would be breaking since none of those versions are currently supported, whereas `>=12` included the supported releases (18 and 20 LTS, 21 current).

Reply:
Potentially - I've pushed an update to the test matrix to test this on `18` and `20`.

I'd been approaching this as:
- rehabilitating the projects for our current minor versions
- declaring support for more recent Nodes in the next major versions

But the project has continued to work on the occasions when I've forgotten to change my version of Node locally.

I don't think that restricting `package.engines.node` in this way will break a build, since `package.enginesStrict` is deprecated now, and isn't declared in our package anyway.  It was intended more as an indication of those versions that this minor version was designed for.  I've taken the same approach for Ubuntu > `20.04` but more implicitly (readme, test matrix).

From here, I could either:

- leave the LTS versions in the test matrix, and loosen up `package.engines.node` again
- or just loosen up the language in the readme to the effect that we're saying this version is designed for Nodes 12, 14, 16, rather than requires them

@aarongoldenthal @jpw

Reply:
[All passing](https://github.com/pa11y/pa11y-ci/pull/213/checks)

Also, while the performance reason given for narrowing the linting step to one OS might not have held up, the duplication 15 times per warning is probably a new reason to split it into its own job. Will do that in a separate PR.

Reply:
Further changes made:

- `lint` step has moved into its own job
- [Node version test matrix](https://github.com/pa11y/pa11y-ci/pull/213/files#diff-1db27d93186e46d3b441ece35801b244db8ee144ff1405ca27a163bfe878957fR24) is now `[12, 14, 16, 18, 20]`
- updated [readme to reflect this](https://github.com/pa11y/pa11y-ci/blob/0f024d4a06e83e1c817cf294cc7f64834b8a6780/README.md#requirements)
- restored `package.lock.engines.node` to `>= 12`